docs/plugins/integrations/authentication.rst

```diff
@@ -0,0 +1,750 @@
+**************************
+Integration authentication
+**************************
+
+.. contents:: Table of contents
```

Comment:
```suggestion
```
I don't think that we need to add a ToC on every page - it takes a lot of space and they are navigable from the menu.